src/OpenTelemetry.Instrumentation.SqlClient/SqlClientMetricsInstrumentationOptions.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace OpenTelemetry.Instrumentation.SqlClient;
+
+/// <summary>
+/// Options for SqlClient metrics instrumentation.
+/// </summary>
+public class SqlClientMetricsInstrumentationOptions
```

Comment:
For this first iteration, let's not include separate metric instrumentation options. The only thing I'd consider here is the `EnableConnectionLevelAttributes`, but I plan on proposing we drop this option altogether in another PR.